.github/workflows/build-nightly.yml

```diff
@@ -84,21 +84,6 @@ jobs:
         run: yarn --ignore-scripts --frozen-lockfile
       - name: build the app
         run: yarn nightly
-      # - name: clean up
-      #   run: npx rimraf ./dist
-      # - name: install and rebuild
-      #   run: |
-      #     yarn --frozen-lockfile
-      #     yarn install-deps
-      # - name: build
-      #   run: |
-      #     set NODE_ENV=production
-      #     node ./tools/main.js build
-      # - name: nightly
-      #   run: |
-      #     set DEBUG=electron-builder
-      #     set NODE_ENV=production
-      #     npx electron-builder --config electron-builder-nightly.yml
```

Comment:
we should not change config in context of a PR that fix a bug / ship a feature.

Reply:
It seems this stuff was commented anyway

Reply:
:beau: